scripts/templates/generate_coverage_report

```diff
@@ -125,24 +129,38 @@ if [[ "$1" == "--codecov" ]]; then
       echo "WARNING: No CODECOV_TOKEN defined. Upload to CodeCov might fail"
    fi
 
-   # Remove other files to be sure CodeCov uses only the coverage.info file
-   rm -f lcov.info
-   remove_gcda_files
-
    # Upload report to CodeCov
+   if [ ! -d build/tmp ]; then
```

Comment:
Style: `if ! [ -d ....`

Reply:
**BUG**: Don't mention the "build" directory like that: there is the `$BUILD` variable specifically for that. See above.
The build directory doesn't have to be `TILCK_ROOT/build`